Makefile

```diff
@@ -13,6 +13,7 @@ MD_DOCS_EXTRAS = $(addprefix $(MD_DOCS_ROOT),README.md LICENSE.md ROADMAP.md CON
 DOCKER_LABEL    = latest
 DOCKER_TAG      = allennlp/allennlp:$(DOCKER_LABEL)
 DOCKER_TEST_TAG = allennlp/test:$(DOCKER_LABEL)
+DOCKER_RUN_CMD  = docker run --rm -v $$HOME/.allennlp:/root/.allennlp
```

Comment:
This works even on the GitHub runners?

Reply:
Well only with our self-hosted runner, since otherwise `$HOME/.allennlp` would be empty